common/src/main/java/org/opensearch/ml/common/transport/agent/MLAgentUpdateInput.java

```diff
@@ -41,20 +42,28 @@ public class MLAgentUpdateInput implements ToXContentObject, Writeable {
     public static final String AGENT_NAME_FIELD = "name";
     public static final String DESCRIPTION_FIELD = "description";
     public static final String LLM_FIELD = "llm";
+    public static final String LLM_MODEL_ID_FIELD = "model_id";
+    public static final String LLM_PARAMETERS_FIELD = "parameters";
     public static final String TOOLS_FIELD = "tools";
     public static final String PARAMETERS_FIELD = "parameters";
     public static final String MEMORY_FIELD = "memory";
+    public static final String MEMORY_TYPE_FIELD = "type";
+    public static final String MEMORY_SESSION_ID_FIELD = "session_id";
```

Comment:
We don't store `session_id` in agent index. Do we need this in update agent API?

Reply:
I see why,  `MLMemorySpec` class has this field. It's ok to keep this. We can refactor together later.